### ISSUE TITLE
Custom 404 page support

### ISSUE THREAD

**Issue by TheDude53** [kzahel/web-server-chrome#182]
There should be custom 404 page support so it doesn't just say "entry not found". It could be useful if people wanted a custom page to go with the rest of their work.

**Comment by kzahel** [comment]
Good idea. Maybe if you have a 404.html in the root directory it would just show that, for example?

**Comment by TheDude53** [comment]
Yes, that could work. I was thinking more along the lines of it being any file with 404 (excluding extension) as the name. It could default to the "entry not found" if there isn't a 404 file, too.

**Comment by TheDude53** [comment]
Yes, that would work. It could just default to the "entry not found" if a 404.html page wasn't found too.

**Comment by ecc521** [comment]
404.html is probably the best idea for a quick implementation, as it is compatible with GitHub pages.
Allowing the user to manually select the file should also work, but risks overcomplicating what is supposed to be a simple, ready to run, server.

**Comment by AGTeeV** [comment]
I can not open the server it's just showing 404

**Comment by ethanaobrien** [comment]
@kzahel I am thinking about implementing this but I need some help. How would I upload the file to the extension to save it as a 404.html? Would I need to store it using local variables or however google chrome apps do it. Also, any ideas on the UI. Reading the 404.html would be easy but saving it is the hard part.

**Comment by ethanaobrien** [comment]
@kzahel @TheDude53 I have built this - Because Kyle doesnt seem to be active on github anymore you can download my archive.zip and load it through `chrome://extensions`
https://github.com/kzahel/web-server-chrome/pull/264

Download updated version: https://drive.google.com/file/d/12Ewzalvx5ZMdleWcGnms4XFnHWC4j65_/view?usp=sharing
